### ISSUE TITLE
Interesting ChromeOS Issue

### ISSUE THREAD

**Issue by arusher999** [Timendus/ticalc.link#28]
I can transfer files to a TI-84 Plus CE and python edition, but when I tried this with a regular TI-84, I was only able to transfer a broken 2048 file, which ended up launching, but didn't really work. After that, nothing happened when I clicked the send button.

**Comment by Timendus** [comment]
Hey there! That's unfortunate! Does the calculator still accept files from other software? What do you mean by "a broken 2048 file"?

**Comment by arusher999** [comment]
Yes the calculator accepts files from ti-connect. The 2048 file launches on-calc, but just doesn't work, the numbers don't show up, only the empty 2048 box.
